NEWS.md

```diff
@@ -1,5 +1,32 @@
 # What's new in `IntervalArithmetic.jl`
 
+## v0.13
+
+### Breaking changes
+- `convert(Interval{T}, x::T)` no longer does any rounding: it creates
+the thin (zero-width) interval `[x, x]`. [#114](https://github.com/JuliaIntervals/IntervalArithmetic.jl/pull/114)
+
+  Note that this implies that numbers are no longer rounded in operations combining them with intervals, such as `0.1 + interval(0.2, 0.3)`, since `0.1` is `convert`ed to an `Interval` by Julia's promotion machinery.
```

Comment:
Auch! I didn't realized this. It seems dangerous, isn't it?
How can this be circumvented?

Reply:
```
julia> (1//10) + (1..2)
Interval(1.0999999999999999, 2.1)
```

Reply:
```
julia> @btime (1//10) + (1..2)
  149.455 ns (4 allocations: 64 bytes)

julia> @btime 0.1 + (1..2)
  25.432 ns (0 allocations: 0 bytes)
```

Reply:
But 
```

julia> 0.1 * (1..2)
Interval(0.1, 0.2)

julia> big(ans)
Interval(1.000000000000000055511151231257827021181583404541015625000000000000000000000000e-01, 2.000000000000000111022302462515654042363166809082031250000000000000000000000000e-01)
```

Reply:
My concern is define a function `f(x) = 0.1 + x` and find out that 0.1 is not in `f(interval(0.0))`...
I guess we should extend solving #120 to other binary operations involving a `Real`.

Reply:
> Basically it's treating 0.1 as the floating-point number 0.1 that it is, and not trying to be "clever" and guessing what the user wants.

These were my thoughts exactly.